src/libraries/System.Diagnostics.EventLog/src/CompatibilitySuppressions.xml

```diff
@@ -1,4 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
```

Comment:
nit: could remove auto-added change this since no other changes here.

Reply:
fixed